### DISCUSSION TITLE
Who is testing OneDrive Client Version v2.5.0-alpha-1 ?

### DISCUSSION THREAD

**Discussion by abraunegg** [abraunegg/onedrive#2486] [Polls]
OneDrive Client Version v2.5.0-alpha-1 has been available as a PR to enable early testing. I am interested trying to know how many people are testing the PR so I can understand how robust the code is. Having just myself run through my scenarios is too weak IMHO.

**Comment by tdcockers** [comment]
I'm not testing at the moment because my singular use case at the moment is Sharepoint syncing. Will find a way to test once that function comes available.

**Reply by abraunegg** [reply]
@tdcockers 
Thanks for the feedback. I am currently working on OneDrive Business Shared Items - and this will drop as alpha-1

SharePoint will then be looked at as part of alpha-2
